dapps/v2Explorer/src/utils/UniversalProvider.tsx

```diff
@@ -3,10 +3,13 @@ import UniversalProvider from '@walletconnect/universal-provider';
 
 // @ts-expect-error - `@env` is a virtualised module via Babel config.
 import {ENV_PROJECT_ID, ENV_RELAY_URL} from '@env';
+import {SessionTypes} from '@walletconnect/types';
+import {ethers} from 'ethers';
+import {Alert} from 'react-native';
 
-export let universalProvider;
-export let currentWCURI;
-export let universalProviderSession;
+export let universalProvider: UniversalProvider;
+export let web3Provider: ethers.providers.Web3Provider | undefined;
+export let universalProviderSession: SessionTypes.Struct | undefined;
 
 export async function createUniversalProvider() {
   console.log('[CONFIG] ENV_PROJECT_ID:', ENV_PROJECT_ID);
```

Comment:
Nitpick: Can we comment out Line 15 / 16 for the console.logs?

Reply:
😆 of course